.grit/patterns/python/cleaned_data_vs_post.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Use `$FORM.cleaned_data[]` instead of `request.POST[]` after `form.is_valid()`
+---
+
+Use `$FORM.cleaned_data[]` instead of `request.POST[]` after `form.is_valid()` has been executed to only access sanitized data.
+
+### references
+- https://docs.djangoproject.com/en/4.2/ref/forms/api/#accessing-clean-data
+
+tags: #fix, #best-practice
+
+```grit
+engine marzano(0.1)
+language python
+
+`if $form.is_valid(): $conditionBody` where {
+    $conditionBody <: contains or {`$request.POST`, `$request.POST.get`} => `$form.cleaned_data`
```

Comment:
Add a test case for `.get` - I don't think this rewrite will work correctly for that.

Reply:
Hi @morgante, 
thanks for pointing out you were correct, it should be `$request.POST` `.get` will be same.

updated the pattern now